lib/ledger_sync/adaptors/quickbooks_online/webhook.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require_relative 'webhook_event'
+
+# ref: https://developer.intuit.com/app/developer/qbo/docs/develop/webhooks/managing-webhooks-notifications#validating-the-notification
+module LedgerSync
+  module Adaptors
+    module QuickBooksOnline
+      class Webhook
+        attr_reader :notifications,
+                    :original_payload,
+                    :payload
+
+        def initialize(payload:)
+          @original_payload = payload
+          @payload = payload.is_a?(String) ? JSON.parse(payload) : payload
```

Comment:
Only comment I have is that I had to pass in my args as `payload.with_indifferent_access`. It could be nice to do that here instead, so the caller doesn't have to.

Reply:
Bump  @ryanwjackson

Reply:
@mattgmarcus `.with_indifferent_access` is Rails (I believe) and not pure ruby.

Reply:
Ah yes, that's right
Okay so I'll just merge